issue_tracker/channels/backends/discord_backend.py

```diff
@@ -0,0 +1,62 @@
+import json
+
+from discord_webhook import DiscordWebhook, DiscordEmbed
+from django.core.exceptions import ImproperlyConfigured
+
+
+class IssueTrackerChannel:
```

Comment:
The class name should be **Channel**. It will be generic and we will use it in all backend. In each backend file naming is fine but class name should be like DiscordChannel(Channel), SlackChannel(Channel)...

Reply:
Changed the class name as per the suggestion.